src/renderer/App.js

```diff
@@ -35,7 +35,7 @@ import { darkTheme } from "../styles/darkTheme";
 import usb from "usb";
 import { withSnackbar } from "notistack";
 
-import KeyboardSelect from "./screens/KeyboardSelect";
+import { KeyboardSelect } from "./screens/KeyboardSelect/index";
```

Comment:
The `index` distinguishes it from the existing KEyboardSelect component currently